CIP-0005/README.md

```diff
@@ -78,6 +78,8 @@ We define the following set of common prefixes with their corresponding semantic
 | `stake_vkh`        | Stake address verification key hash                | blake2b\_224 digest of a delegation verification key   |
 | `stake_shared_vkh` | Shared stake address verification key hash         | blake2b\_224 digest of a delegation verification key   |
 | `vrf_vkh`          | VRF verification key hash                          | blake2b\_256 digest of a VRF verification key          |
+| `datum`            | Output datum hash                                  | blake2b\_256 digest of output datum                    |
+| `redeemer`         | Script data hash                                   | blake2b\_256 digest of script data                     |
```

Comment:
I don't know what this means. Redeemers aren't hashed, I don't think. Do you mean the hash of an actual script? Then that is definitely not a redeemer.

Reply:
I believe this refers to the script integrity hash, doesn't it? In which case (a) `redeemer` is definitely not a good name, and (b) I am not sure we need / want a bech32 prefix for that one. It's very "internal" and not quite relevant to end users.

Reply:
It refers to the `script_data_hash` field in the `tx_body`(see [CDDL](https://github.com/input-output-hk/cardano-ledger/blob/master/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L61)). As it's included in the tx body, it gets processed by HW wallets and should be shown to the user.

Perhaps `script_data` would be a better prefix?

Reply:
I've renamed `redeemer` to `script_data` -> https://github.com/cardano-foundation/CIPs/pull/177/commits/769b7c6595fdad5bd5561bef30cde231af6587f0